### ISSUE TITLE
[recipe]: Minecraft (All Doors only *2)

### ISSUE THREAD

**Issue by wolfkidsounds** [wolfkidsounds/modpack_minecraftnewearth#1]
**Is your request related to a problem? Please describe.**
*A: All doors are produced 3 times when using the recipe, which makes no sense*

---

**Name the Machine/Crafting Unit**
*A: Crafting Table*

**Name the Input Item(s)**
*Shaped*
wood, wood, null
wood, wood, null
wood, wood, null

**Name the Output**
wooden door * 2

**Should the recipe use ore-dictionary?**
*A: Yes*

**Comment by wolfkidsounds** [comment]
Doors ->
```
<minecraft:wooden_door>
<minecraft:spruce_door>
<minecraft:birch_door>
<minecraft:jungle_door>
<minecraft:acacia_door>
<minecraft:dark_oak_door>
<minecraft:iron_door>
<malisisdoors:laboratory_door>
<blockcraftery:editable_door>
<blockcraftery:editable_door_reinforced>
<dimdoors:quartz_door>
<dimdoors:gold_door>
<malisisdoors:wood_sliding_door>
<malisisdoors:iron_sliding_door>
<malisisdoors:factory_door>
<malisisdoors:shoji_door>
<malisisdoors:saloon>
<malisisdoors:hitechdoor>
<moreplanets:cheese_spore_door>
<moreplanets:infected_oak_door>
<moreplanets:alien_berry_oak_door>
<pneumaticcraft:pneumatic_door>
<minecraft:wooden_door>
<minecraft:spruce_door>
<minecraft:birch_door>
<minecraft:jungle_door>
<minecraft:acacia_door>
<minecraft:dark_oak_door>
<minecraft:iron_door>
<malisisdoors:laboratory_door>
<blockcraftery:editable_door>
<blockcraftery:editable_door_reinforced>
<dimdoors:quartz_door>
<dimdoors:gold_door>
<malisisdoors:wood_sliding_door>
<malisisdoors:iron_sliding_door>
<malisisdoors:factory_door>
<malisisdoors:shoji_door>
<malisisdoors:saloon>
<malisisdoors:hitechdoor>
<moreplanets:cheese_spore_door>
<moreplanets:infected_oak_door>
<moreplanets:alien_berry_oak_door>
<pneumaticcraft:pneumatic_door>
```

**Comment by wolfkidsounds** [comment]
Script -> [scripts/mods/minecraft.zs](scripts/mods/minecraft.zs)

**Comment by wolfkidsounds** [comment]
https://github.com/wolfkidsounds/modpack_minecraftnewearth/blob/master/scripts/mods/minecraft.zs

**Comment by wolfkidsounds** [comment]
fixed in #22
